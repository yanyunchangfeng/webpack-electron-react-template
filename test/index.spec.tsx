import { App } from 'src/workbench/electron-render/index';
import { create } from 'react-test-renderer';
import React from 'react';
describe('render App test', () => {
  it('render App correctly', () => {
    const testRenderInstance = create(<App />);
    expect(testRenderInstance.toJSON()).toMatchSnapshot();
  });
});
