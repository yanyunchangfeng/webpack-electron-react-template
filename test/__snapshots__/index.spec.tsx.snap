// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App test render App correctly 1`] = `
<div
  className="d-flex"
>
  <div
    className="windowActionWrap"
  >
    <div
      className="window-left"
    />
    <div
      className="window-right"
    />
  </div>
</div>
`;
